optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -290,13 +295,48 @@ def __call__(self, trial):
             )
         )
 
+        trial.set_user_attr("elapsed_secs", elapsed_secs)
+        trial.set_user_attr("average_iteration_time", average_iteration_time)
+        trial.set_user_attr("step_name", self.step_name)
+        trial.set_user_attr("lgbm_params", json.dumps(self.lgbm_params))
```

Comment:
Very sorry for my late comment. Should we prefix the keys similar to the CMA-ES sampler (`cma:some_key`) as some make little sense without the context of this LightGBMTuner and are quite general English phrases. What do you think?

Reply:
Thank you for letting me know. I'm not aware of it.
I'll like to add a prefix like `lightgbm_tuner:some_key`. If you have any concerns, please let me know.

Reply:
Fixed: https://github.com/optuna/optuna/pull/1032/commits/f904eec30e6cff1b29cc619e6f0ce9f8797707b6

Reply:
Thanks!!